modules/persistent-storage-csi-smb-cifs-limits.adoc

```diff
@@ -0,0 +1,15 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-smb-cifs.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="persistent-storage-csi-smb-cifs-limits_{context}"]
+= Limitations
+
+The following limitations apply to the Common Internet File System (CIFS)/Server Message Block (SMB) Container Storage Interface (CSI) Driver Operator:
+
+* FIPS mode is not supported: 
++
+When Federal Information Processing Standards (FIPS) mode is enabled, the use of md4 and md5 are disabled, which prevents users from using ntlm, ntlmv2, or ntlmssp authentication. Also, signing cannot be used because it uses md5. Any CIFS mount that uses these methods fails when FIPS mode is enabled.
+
+* HTTP proxy configuration is not supported.
```

Comment:
Is this OK?

* Using HTTP proxy configuration to connect to outside of the cluster SMB servers is not supported by the CSI driver.
+
Since CIFS/SMB is a LAN protocol, and though it can be routed to subnets, it is not designed to be extended over the WAN, and does not support HTTP proxy settings.

Reply:
Looks good.